google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/CloneGcpDialog.form

```diff
@@ -51,20 +51,34 @@
       </component>
       <component id="4e988" class="javax.swing.JTextField" binding="directoryName">
         <constraints>
-          <grid row="2" column="1" row-span="1" col-span="3" vsize-policy="0" hsize-policy="6" anchor="8" fill="1" indent="0" use-parent-layout="false">
+          <grid row="3" column="1" row-span="1" col-span="3" vsize-policy="0" hsize-policy="6" anchor="8" fill="1" indent="0" use-parent-layout="false">
             <preferred-size width="150" height="-1"/>
           </grid>
         </constraints>
         <properties/>
       </component>
-      <component id="857e7" class="com.google.cloud.tools.intellij.resources.ProjectSelector" binding="repositoryUrl" custom-create="true">
+      <component id="857e7" class="com.google.cloud.tools.intellij.resources.ProjectSelector" binding="projectSelector" custom-create="true">
         <constraints>
           <grid row="0" column="1" row-span="1" col-span="3" vsize-policy="2" hsize-policy="2" anchor="0" fill="3" indent="0" use-parent-layout="false">
             <minimum-size width="200" height="-1"/>
           </grid>
         </constraints>
         <properties/>
       </component>
+      <component id="1645e" class="javax.swing.JLabel">
+        <constraints>
+          <grid row="1" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <text value="Cloud Repository:"/>
```

Comment:
Rename "Cloud Repository" and "Cloud Project" to "Repository" and "Cloud" in this menu?